### PR TITLE
Adding a Changelog for the schema

### PR DESCRIPTION
**Pull request by floydtree**
#### Description of changes:
1. Adding a changelog.md file with all the changes made since v1.0.0 of OCSF.
2. Bumping the version file to 1.1.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    7. Added `container` and `data` to `response` and `request` objects. #738
+    8. Added `group` to the `api` object. #738
+    9. Added `namespace` to the `resource_details` object. #738
+    10. Added `log_level` to the `metadata` object. #738
+    11. Added `length` to the `http_request` object. #768
+    12. Added `is_exploit_available` to the `vulnerability` object. #777
+    13. Added `domain` attribute to the `group` object. #871
+    14. Adjusted attribute requirements in `dns_query`, `dns_answer` objects. #879
+    15. Added firewall, router, switch, hub to endpoint types. #921
+    16. Added is_vpn attribute to the session object. #922
```
Comment (Aniak5) [inline CHANGELOG.md:98]: Do we want to be consistent and put the attribute and object within upticks? 

"Added `is_vpn` attribute to the `session` object."

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+1. Fixed enumeration item duplication in macOS extension. #926
+
+### Deprecated
+1. Deprecated `cwe_uid` and `cwe_url` attributes and removed from `cve` object. #678
+2. Deprecated `http_status` attribute from `HTTP Activity` event to be replaced by `http_response.code`. #767
+3. Deprecated `finding` object in favor of `finding_info` object. #769
+4. Deprecated `proxy` attribute from the dictionary, in favor of `Network Proxy` profile. #856
+5. Deprecated `group_name` attribute. #873
+6. Deprecated `Security Finding` class to be replaced by the new specific classes according to the use-case: `Vulnerability Finding`, `Compliance Finding`, `Detection Finding`, `Incident Finding`. #877
+7. Deprecated Web Resources Access Activity event class. #890
```
Comment (Aniak5) [inline CHANGELOG.md:111]: Same thing here: 
"Deprecated `Web Resources Access Activity` event class. "

File: CHANGELOG.md (hunk 1 above)
Comment (floydtree) [reply CHANGELOG.md]: Totally, must have missed this one.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    12. Added `is_exploit_available` to the `vulnerability` object. #777
+    13. Added `domain` attribute to the `group` object. #871
+    14. Adjusted attribute requirements in `dns_query`, `dns_answer` objects. #879
+    15. Added firewall, router, switch, hub to endpoint types. #921
+    16. Added is_vpn attribute to the session object. #922
+    17. Added `state` to `network_connection_info` object. #932
+
+### Bugfixes
+1. Fixed enumeration item duplication in macOS extension. #926
+
```
Comment (pagbabian-splunk) [inline CHANGELOG.md:103]: We didn't have a macOS extension in v1.0.0

Comment (floydtree) [reply CHANGELOG.md:103]: Ah that's right, good catch.

**Review by pagbabian-splunk — APPROVED** [review]

**Review by mikeradka — APPROVED** [review]

**Review by zschmerber — APPROVED** [review]
Looks good.

**Pull request merged**